src/portal_visualization/builders/anndata_builders.py

```diff
@@ -239,3 +280,142 @@ def _add_spatial_view(self, dataset, vc):
+            obs_set_paths=self._obs_set_paths,
+            obs_set_names=self._obs_set_names,
+            obs_labels_names=self._obs_labels_names,
+            obs_labels_paths=self._obs_labels_paths,
+            obs_locations_path="obsm/X_spatial",
+            obs_embedding_paths=["obsm/X_umap", "obsm/X_pca"],
+            obs_embedding_names=["UMAP", "PCA"],
+            obs_embedding_dims=[[0, 1], [0, 1]],
+            feature_labels_path="var/hugo_symbol",
+            request_init=self._get_request_init(),
+            initial_feature_filter_path="var/top_highly_variable",
+            coordination_values={
+                "obsType": "cell",
+                "featureType": "gene",
+                "featureLabelsType": "gene",
+            }
+        )
+        dataset = vc.add_dataset(
+            name='Visium'
+        ).add_object(
+            visium_image
+        ).add_object(
+            visium_spots
+        ).add_object(
+            obs_sets
+        )
+        return dataset
+
+    def _setup_anndata_view_config(self, vc, dataset):
+        # Add / lay out views
+        umap = vc.add_view(
+            cm.SCATTERPLOT, dataset=dataset, mapping="UMAP",
+            w=3, h=6, x=0, y=0)
+        spatial = vc.add_view(
+            "spatialBeta", dataset=dataset,
+            w=3, h=6, x=3, y=0)
+        heatmap = vc.add_view(
+            cm.HEATMAP, dataset=dataset,
+            w=6, h=6, x=0, y=6
+        ).set_props(transpose=True)
+
+        lc = vc.add_view("layerControllerBeta", dataset=dataset,
+                         w=6, h=3, x=6, y=0)
+
+        # Add scatterplot, cellsets, gene list, cellsets expression, and heatmap
+        cell_sets = vc.add_view(
+            cm.OBS_SETS, dataset=dataset,
+            w=3, h=4, x=6, y=2)
+
+        gene_list = vc.add_view(
+            cm.FEATURE_LIST, dataset=dataset,
+            w=3, h=4, x=9, y=2)
+
+        cell_sets_expr = vc.add_view(
+            cm.OBS_SET_FEATURE_VALUE_DISTRIBUTION, dataset=dataset,
+            w=6, h=5, x=6, y=7
+        )
+
+        all_views = [spatial, lc, umap, cell_sets, cell_sets_expr, gene_list, heatmap]
+
+        self._views = all_views
+
+        spot_views = [spatial, lc]
+
+        # selected_gene_views = [umap, gene_list, heatmap, spatial]
+
+        # Link spatial view and layer controller
+        vc.link_views_by_dict(all_views, {
+            "spatialTargetZ": 0,
+            "spatialTargetT": 0,
+            "imageLayer": CL([{
+                "fileUid": 'visium',
+                "spatialLayerOpacity": 1,
+                "spatialLayerVisible": True,
+                "photometricInterpretation": 'RGB',
+            }]),
+            "spotLayer": CL([{
+                "obsType": 'spot',
+                "spatialLayerVisible": True,
+                "spatialLayerOpacity": 0.5,
+                "spatialSpotRadius": self._get_scale_factor(),
+                "featureValueColormapRange": [0, 1],
+            }]),
+        })
```

Comment:
```suggestion
        vc.link_views_by_dict(spatial_views, {
            "imageLayer": CL([{
                "photometricInterpretation": 'RGB',
            }]),
        }, scope_prefix=get_initial_coordination_scope_prefix(dataset_uid, 'image'))
        vc.link_views_by_dict(spatial_views, {
            "spotLayer": CL([{
                "spatialLayerOpacity": 0.5,
                "spatialSpotRadius": self._get_scale_factor(),
            }]),
        }, scope_prefix=get_initial_coordination_scope_prefix(dataset_uid, 'obsSpots'))
```

I realized I had to add the `scope_prefix` parameter here https://github.com/vitessce/vitessce-python/pull/325 (now in v3.2.3). This will ensure these values get properly combined with the rest of the automatically-initialized values.